### PR TITLE
Fix heat map labels and prevent image to be draggable

### LINKED ISSUE #228: Remove label on heatmap

### LINKED ISSUE #229: Cover flower image with div (?) to prevent dragging

### PR DESCRIPTION
**Pull request by kome12**
## Description

Fix heat map labels and prevent image to be draggable

## Closes issue(s)

Closes #228 
Closes #229 

## Screenshots

If you have image share...

## Changes include

- [ ] Bugfix (non-breaking change that solves an issue)
- [X] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

## Checklist

- [X] I have tested this code
- [ ] Add here the checks for this PR. If any please delete these checks

### CODE REVIEW

File: src/pages/Florist/index.tsx [hunk 1]
```diff
@@ -262,6 +263,7 @@ export const Florist = () => {
                         src={flower.imageURL}
                         alt={"secret flower pic"}
                         className={styles.notBoughtPic}
+                        draggable="false"
```
Comment (kome12) [inline src/pages/Florist/index.tsx:266]: Set draggable to false.

File: src/pages/MyGrowth/components/TabData.tsx [hunk 2]
```diff
@@ -1,29 +1,37 @@
...
+  format,
   formatISO,
   getDay,
   isBefore,
   isSameDay,
+  isSameMonth,
+  startOfMonth,
   startOfWeek,
   sub,
-  // getMonth,
```
Comment (kome12) [inline src/pages/MyGrowth/components/TabData.tsx:14]: Removed unused import

File: src/pages/MyGrowth/components/TabData.tsx [hunk 3]
```diff
@@ -162,38 +170,37 @@ export const TabData: React.FC<TabDataProps> = ({
 
   const heatMapOptions: ApexOptions = useMemo(() => {
     let date = startOfWeek(startDate);
-    // const startMonth = getMonth(date);
```
Comment (kome12) [inline src/pages/MyGrowth/components/TabData.tsx:165]: Removed commented out line

File: src/pages/MyGrowth/components/TabData.tsx [hunk 4]
```diff
@@ -162,38 +170,37 @@ export const TabData: React.FC<TabDataProps> = ({
...
+    const options: ApexOptions = {
       chart: {
         height: 350,
         type: "heatmap",
-        foreColor: "#172F4A"
+        foreColor: "#172F4A",
       },
       dataLabels: {
-        enabled: true,
+        enabled: false,
```
Comment (kome12) [inline src/pages/MyGrowth/components/TabData.tsx:194]: Set label to false to remove labels from heat map

**Review by conalli — APPROVED** [review]
LGTM!

**Review by curtisahuang — APPROVED** [review]
LGTM

**Review by daiHash — APPROVED** [review]

**Pull request merged**